src/dmd/toir.d

```diff
@@ -961,6 +992,10 @@ void buildClosure(FuncDeclaration fd, IRState *irs)
         //printf("structsize = %d\n", cast(uint)structsize);
 
         Closstru.Ttag.Sstruct.Sstructsize = cast(uint)structsize;
+        fd.csym.Sscope = sclosure;
```

Comment:
What is the current documentation on Sscope?

Change looks fine, just want to make sure that there can be no doubt over what it is used for.

Reply:
> What is the current documentation on Sscope?

It kind of fits:
```
Symbol *Sscope;             // enclosing scope (could be struct tag,
                                // enclosing inline function for statics,
                                // or namespace)
```